README.md

```diff
@@ -262,22 +119,24 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 <!-- FAQ (optional) -->
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
 > Add at least 2 questions new developers would ask when they decide to use your project.
 
-- **[Question_1]**
+- **[What is HTML]**
 
-  - [Answer_1]
+  - [HTML is a hyper text markup language that is used to build raw structure of a web page]
 
-- **[Question_2]**
+- **[What is CSS]**
 
-  - [Answer_2]
+  - [CSS is a cascading style sheets that links to the HTML and make the website visually active.]
 
```

Comment:
- [ ] [OPTIONAL] Like you will agree, the brace bracket is needless here as the reviewer mentioned earlier.  Kindly adjust this to make your work more professional and readable. :wink: